src/core_modules/capture-core/components/DataEntries/TeiRegistrationEntry/hooks/useFormValuesFromSearchTerms.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+import { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { convertClientToForm } from '../../../../converters';
+
+
+export const useFormValuesFromSearchTerms = () => {
```

Comment:
Hey @jasminenguyennn , 

I noticed that for TEAs where the search terms use the `From-To` format, the `[object Object]` value is displayed on the new TEI page. Some examples of these TEAs are the ones with the type `Date` or `Number`.

Maybe these fields should be left empty or pre-filled with either the `from` or `to` search terms?  WDYT? You can find more details in the below video. 


https://user-images.githubusercontent.com/6817073/194064429-665747ca-3647-43a3-aa28-3e0649d00c40.mov


Thank you!